src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -140,6 +140,7 @@ export default class MultipleProjectSelector extends React.PureComponent {
         icon={<StyledInlineSvg src="icon-project" />}
         locked={true}
         lockedMessage={t(`This issue is unique to the ${forceProject.slug} project`)}
+        settingsLink={`/settings/${organization.slug}/${forceProject.slug}/`}
```

Comment:
```suggestion
        settingsLink={`/settings/${organization.slug}/projects/${forceProject.slug}/`}
```
Will save a client-side redirect.

Reply:
good catch!